From 3e6d15e40fc88338482b66eb52920d55bc9a6ca4 Mon Sep 17 00:00:00 2001
From: Alexis López Zubieta <contact@azubieta.net>
Date: Fri, 27 Sep 2019 10:18:54 -0400
Subject: Fix typo

Replace "emtpy" by "empty"

diff --git a/src/DesktopEntry/AST/Group.cpp b/src/DesktopEntry/AST/Group.cpp
index 61c7902..726988a 100644
--- a/src/DesktopEntry/AST/Group.cpp
+++ b/src/DesktopEntry/AST/Group.cpp
@@ -9,7 +9,7 @@ namespace XdgUtils {
             Group::Group(const std::string& headerRawValue, const std::string& headerValue) : headerRawValue(
                 headerRawValue), headerValue(headerValue) {
                 if (headerValue.empty())
-                    throw std::runtime_error("Group Header cannot be emtpy");
+                    throw std::runtime_error("Group Header cannot be empty");
             }
 
             std::string Group::getValue() const {
@@ -18,7 +18,7 @@ namespace XdgUtils {
 
             void Group::setValue(const std::string& newValue) {
                 if (newValue.empty())
-                    throw std::runtime_error("Group Header cannot be emtpy");
+                    throw std::runtime_error("Group Header cannot be empty");
 
                 auto pos = headerRawValue.find(headerValue);
 
